build-tools/packages/build-cli/src/commands/release/report.ts

```diff
@@ -678,5 +678,5 @@ async function writeReport(
     log?.info(`${kind} report written to ${reportPath}`);
     const reportOutput = toReportKind(report, kind);
 
-    return writeJson(reportPath, sortJson(reportOutput), { spaces: 2 });
+    return writeJson(reportPath, reportOutput, { spaces: 2 });
```

Comment:
Is the output no longer sorted?

Reply:
As of now, the output is not sorted. Sorting the `reportOutput` was messing the date.